configuration.default.ini

```diff
@@ -32,6 +32,8 @@ comment = "Hi, I'm here to play radio, local music or youtube/soundcloud music.
 # default volume from 0 to 1.
 volume = 0.1
 stereo = True
+# bandwidth used by the bot
+bandwidth = 200000
```

Comment:
As discussed, can you set de default to 96000 ?

Reply:
Will do!